src/p_mobj.c

```diff
@@ -532,7 +532,7 @@ static void P_ZMovement (mobj_t* mo)
 		// and utter appropriate sound.
 
 		mo->player->deltaviewheight = mo->momz>>3;
-		S_StartSound (mo, sfx_oof);
+		P_HitFloor(mo); // [FG] play sound when hitting animated floor
```

Comment:
GZDoom plays water splash sounds not only when `mo->momz < -GRAVITY*8`, see https://github.com/ZDoom/gzdoom/blob/3d38d5e30cf84186732afd0453e0bd188d94fe99/src/playsim/p_mobj.cpp#L2526

I think water splashes are too rare currently, it feels like the mod is not working.

Reply:
This would need some rework then. I think there are sounds for "big" and "small" splashes in the PWAD, we just need to make sure that only one or the other is played.